### ISSUE TITLE
Can't Install/Uninstall

### ISSUE THREAD

**Issue by rickmcquesten** [eirannejad/pyRevit#1172]
**Describe the bug**
I installed v4.8.5 - my first pyRevit installation. It didn't load in Revit - looked like it was not installed to Revit add-ins folder so I moved files to that folder and now nothing seems to work. I have tried everything I can find here that relates to my issue (seems lots of people have the same issue with not being able to uninstall). Now when I try to install/uninstall I get the message "There is a problem with this Windows Installer package. A program run as part of the setup did not finish as expected. Contact your support personnel or package vendor" I have tried all the regedit.exe file cleanup and tried to delete anything I could find related to pyRevit and it didn't help. I started deleting anything that said "iron python" then started to worry that I shouldn't be doing that? I did find in "HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Installer\Folders" lots of "Iron Python" files, but did not delete all of them because I started to worry that that was not related to pyRevit?? I also tried pyrevit-doctor and nothing seems to happen. Just says "Purgin old installs..." and doesn't seem to do anything.

Any help would be appreciated. Others in my office love pyRevit so I'd like to be able to use it.

I am using Windows 10

I entered command prompt 'pyrevit env' and got a message saying it "is not recognized as an internal or external command, operable program or batch file"

Thanks for your help,

Rick

 - pyRevit Environment: Open a command prompt and run the command below. Replace these lines with the results. Take a look at this markdown guide and wrap the command results in ``` when pasting here for correct formatting.

`pyrevit env`

**Comment by doolmeg** [comment]
I have a similar issue, Ive been using revit2020 for some time now, and my friend suggested me to try out pyrevit.

I downloaded "pyRevit_4.8.5_signed.exe", and run it, after its loaded nothing happens, I opened Revit and nothing pops up. 
I did some research come up with this website --> https://www.notion.so/Install-Errors-f6495adb56254e9e8021f8a68e3b2ab7 

Then I opened cmd and type in "pyrevit env" to check the settings, but still cant solve anything.
The website also suggested to type "pyrevit attach master 277 --installed" into cmd, but cmd reports Error:cannot  find clone 'master'. Been looking through youtube for help as well, but most of the tutorials are in 2017.

Really not sure how should I approach this.

**Comment by rickmcquesten** [comment]
When I run command prompt pyrevit env, it says "'pyrevit' is not recognized as an internal or external command, operable program or batch file."
![cmd](https://user-images.githubusercontent.com/79110480/108377548-02839b80-71b9-11eb-87f9-c903184cf209.jpg)

**Comment by yuvdal** [comment]
I have the same issue. Is there any solution?

**Comment by doolmeg** [comment]
i've tried to install pyrevit on another computer, same version of revit (2020), and same version of pyrevit (pyRevit_4.8.5_signed), and it worked. 
My guess is when i first installed pyrevit, i accidentally clicked (never load), thats why its not showing up anymore. so i reinstalled revit on my old computer, but it didnt worked. Still trying to fingure out how to install it.

**Comment by doolmeg** [comment]
oh, i just found the solution for my old laptop from this website
https://github.com/eirannejad/pyRevit/issues/1075

try go to cmd and type 
pyrevit clones add master "C:\Users\Jose De Jesus\AppData\Roaming\pyRevit-Master"
then
pyrevit attach master 277 --installed

**Comment by eirannejad** [comment]
Glad you fixed the issue. I'll add these to the troubleshooting guides

**Comment by MichaelFrMax** [comment]
Hi all!

I ran into the same problem. I may not have tried in the best way to remove one of the add-ons. But, unfortunately, I came to the same error. 

I've read this thread, but I don't understand well what I should do. 
I tried running the [https://github.com/eirannejad/pyRevit/issues/1075] , but got the same result as rickmcquesten.  I don't understand this branch very well.

Can someone help me, I would be very happy, because I have spent more than one hour trying to solve this problem.
